Assets/_BForBoss/_Character/Scripts/CharacterMovementAudio.cs

```diff
@@ -42,6 +42,13 @@ public void OnUpdate()
                 OnSoundStateChange?.Invoke(_state);
             }
         }
+
+        public float GetPlayerSpeedNormalized()
+        {
+            return _state == MovementSoundState.Running
```

Comment:
I know this might be overkill but I think for safety, we should check that `_characterMovement.CharacterMaxSpeed` != 0. so that we don't get a divide by 0 error!

Reply:
i.e. `Math.max(_characterMovement.CharacterMaxSpeed, 1)`